ADQL.tex

```diff
@@ -1231,7 +1231,7 @@ \subsection{Geometric types}
 ADQL provides support for the POINT, CIRCLE and POLYGON geometric
 types defined in the \DALISpec.
 
-ADQL also provides support for STC-S based geomertic regions,
+ADQL also provides support for STC-S based geometic regions,
 as defined in the \STCSSpec, using the REGION datatype.
```

Comment:
this should be "as defined in \DALISpec"

Reply:
STC-s regions are defined in STC-s...not in DALI, isn't it?
The REGION datatype is an STC-s string, no?

Reply:
Yes, that is the current state, but what I got from the ADASS/IVOA meeting was that ADQL would refer to DALI, and then DALI would in turn refrer to STC-s. This means that if/when DALI reaches agreement on a new form of REGION, then we don't have to update ADQL.

Originally ADQL referred to TAP 1.0, which then referred to STC-s, but when that was removed from TAP it left ADQL hanging. Rather than continuing to refer to an old version of TAP, ADQL should refer to DALI and DALI can decide how to define REGION.